perllib/FixMyStreet/Cobrand/FixMyStreet.pm

```diff
@@ -429,4 +429,11 @@ around 'munge_sendreport_params' => sub {
     $row->areas($original_areas);
 };
 
+sub reopening_disallowed {
+    my ($self, $problem) = @_;
+    my $c = $self->{c};
+    return 1 if $problem->to_body_named("Merton") && $c->user_exists && (!$c->user->from_body || $c->user->from_body->name ne "Merton Council");
+    return super();
```

Comment:
As codecov says, this line isn't covered, and if it were, it would die because super does not exist (eg visiting any non-Merton report on .com). You called `next::method($problem)` fine in the other function.